### PR TITLE
chore (Topology): some universe tweaks

### PR DESCRIPTION
**Pull request by grunweg**
Mostly giving universes explicit/nicer names.
In two files (`ExtremallyDisconnected` and `ProperMap`), remove autoImplicit true.
(My understanding is that this is desired; happy to revert otherwise.)

### CODE REVIEW

File: Mathlib/Topology/Clopen.lean [hunk 1]
```diff
@@ -20,9 +20,7 @@ A clopen set is a set that is both open and closed.
 
 open Set Filter Topology TopologicalSpace Classical
 
-universe u v
-
-variable {α : Type u} {β : Type v} {ι : Type*} {π : ι → Type*}
+variable {α : Type*} {β : Type*} {ι : Type*} {π : ι → Type*}
```
Comment (eric-wieser) [inline Mathlib/Topology/Clopen.lean:23]: arguably a better choice for this type of thing is
```
universe uα uβ uι uπ
variable {α : Type uα} {β : Type uβ} {ι : Type uι} {π : ι → Type uπ}
```

This generates much nicer names than `Type*`, and as a result means that when faced with universe errors, it's much easier to tell what is going wrong.

`Type*` is a great compromise for when you're feeling lazy, but if files already go to the effort of naming universes manually, I'd argue that replacing them with `Type*` is a regression.

Comment (grunweg) [reply Mathlib/Topology/Clopen.lean]: Good point. I pushed a commit addressing this.

File: Mathlib/Topology/Connected.lean [hunk 2]
```diff
@@ -1380,7 +1379,7 @@ theorem isTotallyDisconnected_singleton {x} : IsTotallyDisconnected ({x} : Set 
 #align is_totally_disconnected_singleton isTotallyDisconnected_singleton
 
 /-- A space is totally disconnected if all of its connected components are singletons. -/
-class TotallyDisconnectedSpace (α : Type u) [TopologicalSpace α] : Prop where
+class TotallyDisconnectedSpace (α : Type*) [TopologicalSpace α] : Prop where
   /-- The universal set `Set.univ` in a totally disconnected space is totally disconnected. -/
```
Comment (eric-wieser) [inline Mathlib/Topology/Connected.lean:1383]: These changes save one character in the source in exchange for two more characters (`u_1` and `u`) in the docs and goal view; so I'm not sure they're worthwhile.

**Comment by grunweg** [comment]
Reverted one last round of not-useful changes, squashed and rebased. I am sorry for all the churn.

**Comment by sgouezel** [comment]
I don't think it is useful to give nice names to universes. Quite the opposite in fact: having `Type*` whenever possible is best.

**Comment by grunweg** [comment]
I see. I've heard two contradictory comments now, it's hard to adhere to both...

I don't care about this strongly enough to pursue this further. Feel free to re-use some of this is you do.

**Pull request closed without merging**